client/src/components/questions/QuestionList.jsx

```diff
@@ -0,0 +1,17 @@
+import React, {useState, useEffect} from 'react';
+import Question from './Question.jsx';
+import api from '../../lib/api.js';
```

Comment:
do you need api here? We all, myself included, probably need to scrub our imports to get rid of extraneous stuff.